### ISSUE TITLE
Tables Headers should have scope should be "row" or "col" where appropriate

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#410]

**Comment by msecret** [comment]
Acceptance criteria:

- [ ] The headers on the app general information table need a scope of row added to them

**Comment by jeremiak** [comment]
👍
